streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -302,10 +304,29 @@ public V apply(Change<V> change) {
     public <K1> KTable<K1, Long> count(final KeyValueMapper<K, V, K1> selector,
                                        Serializer<K1> keySerializer,
                                        Serializer<V> valueSerializer,
-                                       Serializer<Long> aggValueSerializer,
                                        Deserializer<K1> keyDeserializer,
                                        Deserializer<V> valueDeserializer,
-                                       Deserializer<Long> aggValueDeserializer,
+                                       String name) {
+
+        return count(
+                selector,
+                keySerializer,
+                valueSerializer,
+                new LongSerializer(),
```

Comment:
Similar to what @ijuma said at https://github.com/apache/kafka/pull/1066, why are we creating new `LongSerializer` and `LongDeserializer` instances for every call to `count()`?  Can't we reuse the same (e.g. cached) instances?